SMLHelper/Handlers/CraftDataHandler.cs

```diff
@@ -26,11 +28,11 @@ private CraftDataHandler()
         /// <para>Can be used for existing TechTypes too.</para>
         /// </summary>
         /// <param name="techType">The TechType whose TechData you want to edit.</param>
-        /// <param name="techData">The TechData for that TechType.</param>
-        /// <seealso cref="TechData"/>
-        public static void SetTechData(TechType techType, ITechData techData)
+        /// <param name="recipeData">The TechData for that TechType.</param>
+        /// <seealso cref="RecipeData"/>
+        public static void SetTechData(TechType techType, RecipeData recipeData)
```

Comment:
**TODO**  Determine if it would be best to have two separate files for each version of `CraftDataHandler` to instead to handle the changes within the same file.

Reply:
A single Handler was maintained with different sections for Subnautica and Below Zero.  
The `ICraftDataHandler` interface was updated to have slightly different methods for the two games.